site/en/blog/service-worker-static-routing-api-origin-trial/index.md

```diff
@@ -0,0 +1,100 @@
+
+As explained, the initial version of the API is essentially an escape hatch from service worker control for some paths. Where this will make sense to use is going to be dependent on how you use your service worker and how users traverse your site.
+
+One example might be if your site uses a [cache-first strategy](https://web.dev/articles/offline-cookbook#cache-falling-back-to-network) (falling back to network), but there's some content that's so rarely visited that there's little to no value in it ever hitting the cache (like archived content or RSS feeds). Restricting these paths to be fetched from the network only _can_ be set up in the service worker, but the service worker still has to start up and run to decide how to handle those requests.
+
+The static routing API, in contrast, bypasses the service worker completely with a few declarative lines:
+
+```js
+self.addEventListener('install', event => {
+  if (event.registerRouter) {
+    event.registerRouter([{
+      condition: {
+        urlPattern: {pathname: '/feeds/*.xml'},
+      },
+      source: 'network',
+    }, {
+      condition: {
+        urlPattern: {pathname: '/archives/*'},
+      },
+      source: 'network',
+    }]);
+  }
+});
+```
+
+As the static routing API evolves, the plan is for this configuration to get more flexible and support more scenarios, like declaratively racing a network fetch and service worker startup. See [the spec explainer's exploration of a potential "final form" of the API](https://github.com/WICG/service-worker-static-routing-api/blob/main/final-form.md) for more details.
+
+## Trying out the Service Worker Static Routing API
+
+The Service Worker Static Routing API is available in Chrome starting in version 116 behind an origin trial, which allows developers to test out the API on their site with real users to measure the effect. See ["Get started with origin trials"](/docs/web-platform/origin-trials/) for background information on origin trials.
+
+{% Aside 'important' %}
+  Unlike other origin trials you may have tried before, this origin trial will only work by putting the origin trial token in an HTTP header (<code>Origin-Trial: TOKEN_GOES_HERE</code>) <em>served on the service worker script</em>. It will not work in a <code>&lt;meta&gt;</code> element with the <code>http-equiv</code> attribute, or as a header on a page's HTML document.
+{% endAside %}
+
+For local testing, the Service Worker Static Routing API can be enabled with a flag at `chrome://flags/#service-worker-static-router`, or by running Chrome from the command like with `--enable-features=ServiceWorkerStaticRouter`.
+
+## Feedback and future directions
+
+The Service Worker Static Routing API is actively being developed and still being shaped. If it seems like it could be useful for you, please try it out via the origin trial and [provide feedback on the API, implementation, and available functionality](https://github.com/WICG/service-worker-static-routing-api/issues).
```

Comment:
```suggestion
The Service Worker Static Routing API is actively being developed and still being shaped. If it seems like it could be useful for you, please try it out via [the origin trial](https://developer.chrome.com/origintrials/#/view_trial/1458040379361198081) and [provide feedback on the API, implementation, and available functionality](https://github.com/WICG/service-worker-static-routing-api/issues).
```